ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/ParticipantIndexerConfig.scala

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils.app
+
+import com.daml.platform.indexer.IndexerConfig
+
+/** Indexer-specific configuration of a participant.
+  *
+  * Parameters that are shared between the indexer and the ledger API server are stored in the parent [[ParticipantConfig]].
+  * Parameters that are shared between all ledger participants are stored in the parent [[Config]]
+  */
+final case class ParticipantIndexerConfig(
+    allowExistingSchema: Boolean,
```

Comment:
What does `allowExistingSchema` mean?

Reply:
Whether the flyway database migrations will run on a non-empty database without a flyway version history. Normally the indexer should abort if it detects a database schema that was not created by the same flyway version history, to prevent modifying the wrong database.